README.md

```diff
@@ -422,6 +422,9 @@ task_definition:
   services:
     <service_name>:
       essential: boolean
+      cpu_shares: integer
+      mem_limit: integer
+      mem_reservation: integer
```

Comment:
Missed this the firs time -- I believe these take memory strings? Since these are tricky, might be good to explain what formats are supported and perhaps include an example `ecs-params.yml` file for a composeV3 scenario.